### ISSUE TITLE
HW-5 Problem 2

### ISSUE THREAD

**Issue by StuffnThingz** [Montana-Media-Arts/441-WebTech#13]
Can someone help me get started on problem two? I'm confused about how this question is worded. Would set, add, and delete be objects? Do I add them in a constructor with this dot?

**Comment by fbrown222** [comment]
Nope, set, add, and delete would be your methods (think thats the right terminology...). Anyways, they are all part of one object, they are not separate objects.
I think you need this. for your constructor parameter, but I don't think you need it for you individual methods...it might also depend on how you're going about your code...
Gosh I hope I'm using the right terms here. Please, @michaelmusick, correct me if I'm wrong, or anyone for that matter.

**Comment by michaelmusick** [comment]
you are using the right terms! 

Keep at it everyone, you are doing a nice job working through the problems and helping each other!

**Comment by StuffnThingz** [comment]
K so it would look something like:
class Group {
 constructor() {
this.member = [];
}
add(value){}

Then I would add delete and has? Is that a start?

**Comment by ellenkuehl** [comment]
I have something similar to you StuffnThingz! Let me know if I can help at all, although the assignment is due soon!
